molecularnodes/io/wwpdb.py

```diff
@@ -34,8 +34,9 @@ def fetch(
         build_assembly=build_assembly
     )
 
-    model.mn['pdb_code'] = pdb_code
-    model.mn['molecule_type'] = 'pdb'
+    # these lines are throwing errors for me when trying to run tests locally
+    #model.mn['pdb_code'] = pdb_code
```

Comment:
These need to be uncommented. I'll add more details in a comment on the PR explaining what is happening.

Reply:
Should these lines be: 
```
molecule.object.mn['pdb_code'] = pdb_code
molecule.object.mn['molecule_type'] = 'pdb'
```

Also, that last line is incorrect if the `format` keyword arg for the `fetch` function is anything other than `'pdb'`, right?